### DISCUSSION TITLE
Theemostat ESP HomeSpan

### DISCUSSION THREAD

**Discussion by lpuntaloro** [HomeSpan/HomeSpan#486] [General]
Hi, 
I would like to make a thermostat with homespan.. has anyone tried? Do you have a sketch for arduino? Thank you

**Comment by HomeSpan** [comment]
See the HomeSpanReferenceSketches repository.  It contains a complete thermostat example.

**Reply by lpuntaloro** [reply]
Thank you! But do you know that I can't find it?

**Reply by HomeSpan** [reply]
Go to the main page for HomeSpan.  Link is listed there.
